app/views/root/index.html.erb

```diff
@@ -342,7 +342,7 @@
 
 
     <div class = "mt-5 column d-flex flex-column align-items-center justify-content-center">
-      <a href="https://rubyforgood.org/" target="_blank" class="btn btn-outline-secondary px-8 mb-2 text-primary"><%= t('.learn_more_about')%></a>
+      <a href="https://rubyforgood.org/" target="_blank" class="btn btn-primary px-8 mb-2"><%= t('.learn_more_about')%></a>
```

Comment:
Can we do `btn-outline-primary`? It's closer to the original design.

```suggestion
      <a href="https://rubyforgood.org/" target="_blank" class="btn btn-outline-primary px-8 mb-2"><%= t('.learn_more_about')%></a>
```

Reply:
Another option if we want to keep the original design is to leave the text-primary class but remove it with `onmouseover` event. My vote is keeping it simple with `btn-outline-primary` but both work. Thoughts @kasugaijin ?

```suggestion
      <a href="https://rubyforgood.org/" target="_blank" class="btn btn-outline-secondary px-8 mb-2 text-primary"  onmouseover="this.classList.remove('text-primary')"  onmouseout="this.classList.add('text-primary')><%= t('.learn_more_about')%></a>
```
[Screencast from 2024-12-08 09-50-41.webm](https://github.com/user-attachments/assets/e757aaf8-1993-43ab-a5ec-2e70e2bca057)

Reply:
Yeh let’s keep it simple with the btn outline primary class!

Reply:
Choose btn-outline-primary and addressed comments